roles/servicetelemetry/templates/manifest_alertmanager.j2

```diff
@@ -1,8 +1,9 @@
-apiVersion: monitoring.coreos.com/v1
+apiVersion: {{ prometheus_operator_api_string }}/v1
 kind: Alertmanager
 metadata:
   labels:
     alertmanager: '{{ ansible_operator_meta.name }}'
+    {{ prometheus_operator_label }}
```

Comment:
Does this leave an empty line (and does it even matter?) when the label is undefined in a use_community setup? I'm guessing you've run this locally and it didn't care though.

Reply:
I think the answer is that yes it does leave a line, but no it doesn't matter. I'm working on testing all three modes now.